<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClient</class>
 <widget class="QDialog" name="AddClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Contact Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="lbFirstName">
             <property name="text">
              <string>First name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbLastName">
             <property name="text">
              <string>Last name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="txtFirstName">
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtLastName">
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="lbProvince">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Province</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>City</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="cbProvince">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Alberta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>British Columbia</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Manitoba</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Brunswick</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Newfoundland and Labrador</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Northwest Territories</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nova Scotia</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nunavut</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ontario</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prince Edward Island</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quebec</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sakatchewan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yukon</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtCity">
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Home address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtAddress"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Phone #</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="cbAge"/>
           </item>
           <item>
            <widget class="QSpinBox" name="sbAreaCode">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbPhone">
             <property name="minimum">
              <number>1000000</number>
             </property>
             <property name="maximum">
              <number>9999999</number>
             </property>
             <property name="value">
              <number>9999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lbEmail">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtEmail"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pbSubmit">
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbExit">
             <property name="text">
              <string>Exit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
